lib/cylc/gui/gsummary.py

```diff
@@ -92,7 +91,12 @@ def get_task_cycle_statuses(host, suite, owner=None):
         task_order = task.split(".")
         task_order.append(status)
         task_cycle_statuses.append(tuple(task_order))
-    return task_cycle_statuses
+    for line in stdout.split("\n"):
```

Comment:
You can search the whole multiline text (time + bracketed part) by using [re.M](https://docs.python.org/2/library/re.html#re.M)